### PR TITLE
Update UG - Add before and after steps and images

### PR DESCRIPTION
**Pull request by jonahtanjz**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -233,12 +235,21 @@ You can then access the food item later on for editing, deleting etc.
...
+2. Press `Enter` to execute.
+
+Before the `add` command is executed:
 
 ![Before executing the add command](images/AddCommandBefore.png)
 
 <p style="text-align: center; text-decoration: underline">Figure 5: The red box shows the add command being typed into the command box.</p>
 
+<br><br>
+Outcome:
```
Comment (songyiang) [inline docs/UserGuide.md:249]: Is there a need for "Outcome:"? Because the next line seems to be more descriptive. Same for the others.

**Review by rolandyuwy — APPROVED** [review]
LGTM

**Review by songyiang — COMMENTED** [review]
LGTM

File: docs/UserGuide.md (hunk 1 above)
Comment (rolandyuwy) [reply docs/UserGuide.md:249]: To be discussed in meeting

**Pull request merged**